source/nodejs/tests/unit-tests/tsconfig.json

```diff
@@ -1,15 +1,9 @@
 {
+	"extends": "../../tsconfig.json",
+
 	"compilerOptions": {
-		"module": "commonjs",
-		"target": "es5",
-		"lib": [
-			"es2015"
-		],
         "outDir": "./lib",
-		"moduleResolution": "node",
```

Comment:
I strongly suggest all new projects written in TypeScript use these two compiler directives:
```json
"noImplicitAny": true,
"strictNullChecks": true
```